### ISSUE TITLE
[Bug] Amazon Web Services does not show an icon.

### ISSUE THREAD

**Issue by tur1ngb0x** [ente-io/ente#3867]
### Description

Amazon Web Services entry in the Windows Desktop app does not show an icon.

OS: Windows 10 Build 19045.4894
Client: Ente Auth v4.0.2
Service: Amazon Web Services
Issuer: aws.amazon.com
Account: aws.amazon.com

| Main Page  | Token Options |
| ------------- | ------------- |
| ![1](https://github.com/user-attachments/assets/e23c0f41-7707-42a8-acaa-1924aa863200)  | ![2](https://github.com/user-attachments/assets/c9b1fa59-90ad-4089-983f-fa29535522a3)  |

### Version

4.0.2

### What product are you using?

Ente Auth

### What platform are you using?

Desktop - Windows

**Comment by LeGenDaRy15PT** [comment]
Hello @tur1ngb0x 

It will probably be added in the next update, I added this icon last week.
[AWS icon](https://github.com/ente-io/ente/blob/5bf1c4e14fd85d2196cf02a84b3d48866bf55b81/auth/assets/custom-icons/icons/Amazon%20Web%20Services.svg)

**Comment by KingLuc12** [comment]
The issue is the "Account Issuer" isn't "Amazon Web Services". aws.amazon.com has not been added as one of the "altNames".

If you set the issuer name to Amazon Web Services, then the icon should appear. I recommend someone adds "aws.amazon.com" in the list of "altNames" so that by default people will have the logo - Well at least it should work if someone did that but, altNames are currently not working as I mentioned here: #3875 

TLDR: Change the issuer name to "Amazon Web Services" and it should work as expected.

**Comment by LeGenDaRy15PT** [comment]
I actually made a python script to send the name of the images.svg to a notepad and then through that notepad I had each image name per line and as I put in several icons I configured them alphabetically by the “title” tag in “custom-icons.json” because doing it manually would take too long, so it did everything automatically using the script.

I'm still not sure how the “altNames” thing works, but if there's a problem in the future, just put the right name there if it's not recognized as it is.

![imagem](https://github.com/user-attachments/assets/3c016622-6dbf-4d3b-9637-4f4d92cee77b)

I did it like this:
1- Names image per line 
![imagem](https://github.com/user-attachments/assets/03b1b98b-b4d5-497d-85a4-43112e5330cf)
2- Original Code
![imagem](https://github.com/user-attachments/assets/a8a343b9-5dff-4259-846d-4a4e98a8ee19)
3- Modify Code
![imagem](https://github.com/user-attachments/assets/4b8d3744-4abf-4433-824a-dc01cfab7225)
